src/frontend/screens/Library/components/GameCard/index.tsx

```diff
@@ -126,6 +126,25 @@ const GameCard = ({
     updateGameInfo()
   }, [status])
 
+  async function handleUpdate() {
+    return updateGame({ appName, runner, gameInfo })
+  }
+
+  useEffect(() => {
+    const handleAutoUpdate = async () => {
+      if (!hasUpdate) {
+        return
+      }
+      const settings = await window.api.requestAppSettings()
+      const gameSettings = await window.api.requestGameSettings(appName)
+      // if has update check for the AutoUpdateGames setting and if true, call handleUpdate
+      if (settings.autoUpdateGames && !gameSettings.ignoreGameUpdates) {
+        handleUpdate()
+      }
+    }
+    handleAutoUpdate()
+  }, [hasUpdate])
```

Comment:
should this be handled in the backend instead? this will do this check every time we open the library

I think it makes more sense to handle this feature when we check for all game updates at `ipcMain.handle('checkGameUpdates'` in main.tsx instead

Reply:
Hmm, we check only if a game has Update, but I will check if it is possible to do that from the backend only.

Reply:
done